### PR TITLE
Rouge last details

### PR DESCRIPTION
**Pull request by gucifer**
Added a new test for multireference value check.
Added RougeN and RougeL to docs.

Check list:

- [x] New tests are added (if a new feature is added)
- [x] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (sdesrozis) [inline .vscode/settings.json:1]: Please remove this file

File: ignite/metrics/rouge.py [hunk 2]
```diff
@@ -59,15 +58,15 @@ def lcs(seq_a: Sequence[Any], seq_b: Sequence[Any]) -> int:
     return dp[m][n]
 
 
-class Score(namedtuple("Score", ["match", "candidate", "reference"])):
+class _Score(namedtuple("_Score", ["match", "candidate", "reference"])):
```
Comment (sdesrozis) [inline ignite/metrics/rouge.py:61]: I don't understand why you pass this argument in private. It is used as output of public functions.

Comment (gucifer) [reply ignite/metrics/rouge.py:61]: Either we make it private or provide documentation for it. If you want I can write docs for it.

**Review by vfdev-5 — APPROVED** [review]
LGTM!

**Pull request merged**